lib/dgram.js

```diff
@@ -344,6 +346,14 @@ Socket.prototype.send = function(buffer,
   if (typeof callback !== 'function')
     callback = undefined;
 
+  if (typeof address === 'function') {
+    callback = address;
+    address = undefined;
+  } else if (address && typeof address !== 'string') {
+    throw new TypeError('Invalid arguments: address must be a nonempty ' +
+      'string or falsy');
```

Comment:
(Feel free to ignore this. It isn't really about this PR. It's answering an @-mention, that's all.)

@mscdex There is a lint rule checks that arguments are aligned. That's one argument broken across multiple lines so the rule doesn't kick in. If we enforced alignment in that scenario, it would have all sorts of undesirable consequences. (Consider an inline callback. The body of the function would need to be aligned with other arguments before the callback.)

All of this would be more manageable if we either had more lax rules around line length (something I would support--I think breaking a string apart like that does more harm to readability than letting it run off the right side of your editor) or if we didn't enforce alignment and instead had a constant indent (also something I would support), but those ships have probably sailed.